pyproject.toml

```diff
@@ -95,10 +95,6 @@ replacement = 'src="https://raw.githubusercontent.com/encode/httpx/master/\1"'
 [tool.ruff]
 select = ["E", "F", "I", "B", "PIE"]
 ignore = ["B904", "B028"]
-line-length = 88
```

Comment:
Do we need mention the default value?

Reply:
Explicit better than implicit

Reply:
I didn't find in Ruff's docs that prefers default values to use explicitly.
As this PR also removes the `max-line-length` so there aren't any line configs at all.